### DISCUSSION TITLE
依依神奇的脑回路 | /posts/yiyi-talking/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#607] [Announcements]
https://yuanfan.rbind.io/posts/yiyi-talking/

**Comment by residualsun1** [comment]
> 做一朵无忧无虑的云，伤心了就下雨，飘在天上不需要想任何事情。

这句讲得真好。想起了王小波在《黄金时代》里写的一段话：

> “那一天我二十一岁，在我一生的黄金时代，我有好多奢望。我想爱，想吃，**还想在一瞬间变成天上半明半暗的云**，后来我才知道，生活就是个缓慢受锤的过程，人一天天老下去，奢望也一天天消失，最后变得像挨了锤的牛一样。可是我过二十一岁生日时没有预见到这一点。我觉得自己会永远生猛下去，什么也锤不了我。”

**Reply by earfanfan** [reply]
依依的二十二岁生日下个月才到，现在还算二十一岁，她说她过二十一岁生日的时候就知道自己不会永远生猛下去。

我以前听到王小波的名字是读高中的时候，语文老师在课堂上给我们介绍《一只特立独行的猪》，当时一点也没听懂，于是王小波给我留下的印象就是读不懂。现在过了十几年，看到你引用的这段话，很有共鸣，我对这位作家的印象终于改变，我想以后可能会读读他写的文章了。

**Reply by LFG12** [reply]
依依姐和袁凡姐一样，都是十分有趣的灵魂呀，俺现在回到武汉了YF姐，最近在准备深圳大学的考研复试，等三月下旬复试结束后，五一期间目前也不准备回家，有时间可以一起见见面呀

**Reply by earfanfan** [reply]
可以呀，那等你复试结束后我找个周末去找你玩耍，就如你所说的逛逛你们学校，爬一下九真山。依依现在可惨了，总是要加班，就连周六也要，她可能只有端午放假的时候能来看看我了。

**Comment by earfanfan** [comment]
@LFG12 

小朋友，你好，复试怎么样啦，应该顺利通过了吧。撒时候有时间实践一下咱们的约定呢？我本来想去你的博客留言，结果找了一圈还是没找到能留言的框框，所以写在这里了。

**Reply by LFG12** [reply]
嗯嗯，顺利拟录取了袁凡姐姐，在深圳呆了5天后又去在广州玩了两天，昨天刚回到武汉，下午和同学出去转了转，抱歉因当时没有电脑，手机没有登录github不能回复留言，今天上午才在电脑上回复，现在已经回到了学校，期待袁凡姐姐有时间过来玩呀（握爪）
